setup.py

```diff
@@ -1,6 +1,6 @@
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
-version = '0.1.2'
+version = "0.1.3"
```

Comment:
Should we bump this version or leave as is?

Reply:
maybe later when we want to publish new version like in https://github.com/zendesk/txconnpool/commit/1892116782cce13f4ec2b8b60731cbd1b5492260